solidity/contracts/token/libs/MovableCollateralRouter.sol

```diff
@@ -67,44 +35,79 @@ abstract contract MovableCollateralRouter is Router {
         _;
     }
 
-    function addRebalancer(address rebalancer) external onlyOwner {
-        _allowedRebalancers.set(rebalancer, 1);
+    modifier onlyAllowedBridge(uint32 domain, ValueTransferBridge bridge) {
+        EnumerableSet.AddressSet storage bridges = _allowedBridges[domain];
+        require(bridges.contains(address(bridge)), "MCR: Not allowed bridge");
+        _;
     }
 
-    function removeRebalancer(address rebalancer) external onlyOwner {
-        _allowedRebalancers.remove(rebalancer);
+    function rebalancers() external view returns (address[] memory) {
```

Comment:
Nit: Change this to `allowedRebalancers` or change the storage variable to `_rebalancers`